test/integration2/util/knex-instance-provider.js

```diff
@@ -0,0 +1,143 @@
+const { promisify } = require('util');
+const knex = require('../../../lib');
+
+const Db = {
+  PostgresSQL: 'postgres',
+  MySQL: 'mysql',
+  MySQL2: 'mysql2',
+  MSSQL: 'mssql',
+  SQLite: 'sqlite3',
+  Oracle: 'oracle',
```

Comment:
is oracle nowadays oracle and not oracledb ?

Reply:
You are right, that was a mistake on my part. Fixed.